code/modules/surgery/organs/stomach.dm

```diff
@@ -144,7 +144,11 @@
 
 /obj/item/organ/stomach/ethereal/on_life()
 	..()
-	adjust_charge(-ETHEREAL_CHARGE_FACTOR)
+	if(owner.radiation)
+		owner.radiation = max(0,owner.radiation-(ETHEREAL_CHARGE_FACTOR*ETHEREAL_RAD_MULT))
```

Comment:
What is the point of this?

Reply:
they don't consume charge as long as they're irradiated. It's partially to limit proc calls.

Reply:
I was referring to line 148. All you're doing is subtracting by 8 so I'm kinda confused as to what that will accomplish given how high rads can get.

Reply:
oh it's to lower their rads level over time so rads aren't a death sentence